int32 bus_id
int32 demand_change